### PR TITLE
Login

### PR DESCRIPTION
**Pull request by MadayAlcala**
Hi, I solved the comments, please review it.

### CODE REVIEW

File: src/test/java/steps/LoginSteps.java [hunk 1]
```diff
@@ -13,26 +15,24 @@
...
     public void initializeLoginPage() {
-        loginPage = new LoginPage();
+        loginPage = PageTransporter.getInstance().goToUrlLogin();
     }
 
     /**
      * This method fills in the user data to be able to log in.
      *
-     * @param email 'email', represents the email of an user.
+     * @param email    'email', represents the email of an user.
```
Comment (JosueRodriguezGarcia) [inline src/test/java/steps/LoginSteps.java]: Please remove space in this line.

File: automationpractice.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#user credentials
+email = smilingly3@gmail.com
```
Comment (JosueRodriguezGarcia) [inline automationpractice.properties]: please, remove credentials private.

Comment (MadayAlcala) [reply automationpractice.properties]: Solved.

File: src/test/java/steps/LoginSteps.java (hunk 1 above)
Comment (MadayAlcala) [reply src/test/java/steps/LoginSteps.java]: Solved.

**Review by JosueRodriguezGarcia — APPROVED** [review]

**Pull request merged**